app/controllers/albums_controller.rb

```diff
@@ -1,214 +1,89 @@
+
+  def show
     if (@album.images)
       @images = @album.images.order('foto_file_name asc')
     else
-      @images = nil      
-    end    
+      @images = nil
+    end
   end
-  
+
   def new
-    @album = Album.new
   end
-  
+
   def create
-      @album = Album.new(album_params)      
-      @album.update(author: current_user.profile)    
-      respond_to do |format|
-        if @album.save                        
-          format.html { redirect_to @album, notice: 'Albumet skapades!' }
-          format.json { render :json => @album, :status => :created, :location => @album }
-        else
-          format.html { render action: "new" }
-          format.json { render json: @album.errors, status: :unprocessable_entity }
-        end
-      end
+    @album.author = current_user.profile
+    if @album.save
+      redirect_to @album, notice: 'Albumet skapades!'
+    else
+      render action: "new"
+    end
+
   end
-    
-  def destroy    
+
+  def destroy
     @album.destroy
-    respond_to do |format|
-      format.html { redirect_to albums_url,notice: 'Albumet raderades.' }
-      format.json { head :no_content }
-    end
+    redirect_to albums_url, notice: 'Albumet raderades.'
   end
+
   def destroy_images
     for image in @album.images
       image.destroy
     end
-    respond_to do |format|
-      format.html { redirect_to @album, notice: 'Bilderna tog borts!' }
-      format.json { render :json => @album, :location => @album }
-    end
+    redirect_to @album, notice: 'Bilderna tog borts!'
   end
+
   def upload_images
     if (params[:fotos]) && (params[:subcategory_id])
-        #===== The magic is here ;)
-        @count = 1
-        @total = params[:fotos].count
-          params[:fotos].each { |foto|
-            flash[:notice] =("Laddar upp "+@count.to_s+"/"+@total.to_s)
-            @album.images.create(foto: foto,subcategory_id: params[:subcategory_id])
-            @count = @count+1;
-          }
-          flash[:notice] =("Färdig!   Laddat upp "+@total.to_s+" bilder.")
-    end    
+      #===== The magic is here ;)
+      @count = 1
+      @total = params[:fotos].count
+      params[:fotos].each { |foto|
+        flash[:notice] = %(Laddar upp #{@count}/#{@total})
+        @album.images.create(foto: foto, subcategory_id: params[:subcategory_id])
+        @count = @count+1;
+      }
+      flash[:notice] = %(Färdig! Laddat upp #{@total} bilder.)
+    end
   end
+
   def update
-    respond_to do |format|
-      if @album.update_attributes(album_params)                
-        format.html { redirect_to edit_album_path(@album), :notice => 'Albumet uppdaterades!' }
-        format.json { head :no_content }
-      else
-        format.html { render :action => "edit" }
-        format.json { render :json => @album.errors, :status => :unprocessable_entity }
-      end
+    if @album.update(album_params)
+      redirect_to edit_album_path(@album), notice: 'Albumet uppdaterades!'
+    else
+      render action: "edit"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.